.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: win
   pool:
-    vmImage: windows-2022
+    vmImage: windows-2019
```

Comment:
There's a way to make this persist across rerenders, if you add the following to the cbc:
```

azure:
  settings_win:
    pool:
      vmImage: windows-2019
```